conceptual/EFCore.PG/mapping/translations.md

```diff
@@ -151,7 +151,7 @@ Regex.IsMatch(stringValue, "^A+", regexOptions)               | [stringValue ~ '
 The below translations provide functionality for determining the similarity of alphanumeric text based on trigram matching, using the [`pg_trgm`](https://www.postgresql.org/docs/current/pgtrgm.html) extension which is bundled with standard PostgreSQL distributions. All the below parameters are strings.
 
 > [!NOTE]
-> To use these translations, your project must depend on the [Npgsql.EntityFrameworkCore.PostgreSQL.Trigrams](https://www.nuget.org/packages/Npgsql.EntityFrameworkCore.PostgreSQL.Trigrams/) package, and call <xref:Microsoft.EntityFrameworkCore.NpgsqlTrigramsDbContextOptionsBuilderExtensions.UseTrigrams%2A> in your `OnModelConfiguring`.
+> Prior to version 6.0, to use these translations, your project must depend on the [Npgsql.EntityFrameworkCore.PostgreSQL.Trigrams](https://www.nuget.org/packages/Npgsql.EntityFrameworkCore.PostgreSQL.Trigrams/) package, and call <xref:Microsoft.EntityFrameworkCore.NpgsqlTrigramsDbContextOptionsBuilderExtensions.UseTrigrams%2A> in your `OnModelConfiguring`.
```

Comment:
Didn't you integrate `Trigrams` package into the main one?

Reply:
Oh wait, that's only for the previous versions, my bad 😢

Reply:
Yep :)